crates/uv-resolver/src/resolution.rs

```diff
@@ -505,6 +506,8 @@ pub struct DisplayResolutionGraph<'a> {
     /// Whether to include annotations in the output, to indicate which dependency or dependencies
     /// requested each package.
     include_annotations: bool,
+    /// Whether to include indices in the output, to indicate which index was used for each package.
+    include_indices: bool,
```

Comment:
I know it's tedious but I'm going to change this to `indexes` to match the terminology that pip uses in its documentation.

Reply:
Thanks!